.github/workflows/release-v2.yml

```diff
@@ -57,14 +57,13 @@ jobs:
     steps:
       - name: Create release
         id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v1
```

Comment:
updating to a supported action. it uses a variable `token` instead of the environment, and `name` property instead of `release-name`.